internal/objects/crdrules.go

```diff
@@ -178,7 +178,7 @@ func (c *CRDLister) UpdateFQDNHostruleMapping(fqdn string, hostrule string) {
 func (c *CRDLister) GetFQDNFQDNTypeMapping(fqdn string) (bool, string) {
 	found, fqdnType := c.FqdnFqdnTypeCache.Get(fqdn)
 	if !found {
-		return false, ""
+		return true, string(akov1alpha1.Exact)
```

Comment:
If we are always returning true, then probably we dont need to return it.

Reply:
makes sense. done thanks